lighthouse-core/audits/accessibility/aria-allowed-attr.js

```diff
@@ -37,24 +35,6 @@ class ARIAAllowedAttr extends Audit {
       requiredArtifacts: ['Accessibility']
     };
   }
-
-  /**
-   * @param {!Artifacts} artifacts
-   * @return {!AuditResult}
-   */
-  static audit(artifacts) {
```

Comment:
I'm kinda thinking it'd be useful to leave a comment directing the reader:
```js
// see base class in axe-audit.js for audit() implementation
```

but curious what others think